Makefile

```diff
@@ -1,13 +1,34 @@
-.PHONY: com.docker.slirp.exe install uninstall
+.PHONY: com.docker.slirp.exe com.docker.slirp install uninstall
+
+TARGETS :=
+ifeq ($(OS),Windows_NT)
+	TARGETS += com.docker.slirp.exe
+else
+	TARGETS += com.docker.slirp
+endif
+
+all: $(TARGETS)
 
 EXEDIR=C:\projects\vpnkit
 
 com.docker.slirp:
-	cd src/com.docker.slirp && \
-	oasis setup && \
-	./configure && \
-	make && \
-	cp _build/src/main.native com.docker.slirp
+	$(MAKE) -C src/com.docker.slirp
+
+depends:
+	opam repo list | grep com-docker-slirp || opam repo add com-docker-slirp .
```

Comment:
The build at the moment is quite difficult, and perhaps we're stuck choosing between a complicated build, where many steps are needed (as @samoht noted in #24) and an easy / magical build that's too magical because of its side-effects. Longer term things should simplify as we remove the pins. In the meantime is there anything sensible we could do or should we leave it how it is? In practice I'm not going to be able to look at it for a few weeks so perhaps I'll close this and make space for someone to suggest something better.